dailymail/dailymail_stack.py

```diff
@@ -7,38 +7,40 @@
     aws_events as events,
     aws_iam as iam,
     aws_s3,
-
+    aws_s3_deployment,
+    CfnParameter,
 )
 from constructs import Construct
 
 
-class DailymailStackTest(Stack):
+class DailymailStack(Stack):
 
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
-
-        scheduled_lambda = lambda_.Function(self, "MyDailyMailTest",
+        email = CfnParameter(self, "email", type="string", description="email address")
+        scheduled_lambda = lambda_.Function(self, "MyDailyMail",
                                             handler='lambda_function.lambda_handler',
                                             runtime=lambda_.Runtime.PYTHON_3_8,
                                             code=lambda_.Code.from_asset('lambda'),
                                             )
+        bucket = aws_s3.Bucket(self, "TestBucket", bucket_name="testbucketcdk1241210",)
```

Comment:
I see the bucket_name is reused bellow. Either create a constant for it or reference it from the bucket object when you use it on line 45.